daemon/oci_linux.go

```diff
@@ -156,10 +156,10 @@ func WithApparmor(c *container.Container) coci.SpecOpts {
 }
 
 // WithCapabilities sets the container's capabilties
-func WithCapabilities(c *container.Container) coci.SpecOpts {
+func WithCapabilities(c *container.Container, defaultCapabilities []string) coci.SpecOpts {
```

Comment:
Can you add a test for this?

Reply:
No problem, I just wanted to make sure we were on the same page as to the overall approach.

Reply:
Yep, looks great!

Reply:
I added a tests, but the tests are failing locally with a really arcane error when I do a sudo make test, I am going to see what jenkins does with it.